.github/workflows/main.yml

```
@@ -92,17 +93,20 @@ jobs:
   test-linux-source-distribution:
     needs: create-source-distribution
     runs-on: ubuntu-latest
+    # Build & test job should be run on:
+    # - All Ubuntu LTS supported by Zivid
+    # - All Ubuntu versions newer than the latest LTS
```

Is it hard to change this to the following?

```suggestion
    # - 4 latest Ubuntu versions
```

The reason I ask is that it is highly likely that we got users on any recent Ubuntu, we do not want to leave them behind immediately when a new LTS is out.

---

I'll try. Could be easy.

---

Wait Ubuntu 21.04 and 21.10 are already past end-of-life. The apt repos aren't even up anymore.

---

Ok, resolving this thread then.